qemu/tests/timedrift.py

```diff
@@ -125,8 +130,8 @@ def restore_cpu_affinity(prev_masks):
                                              time_format)
 
             # Run some load on the guest
-            for load_session in guest_load_sessions:
-                load_session.sendline(guest_load_command)
+            for i, load_session in enumerate(guest_load_sessions):
+                load_session.sendline(guest_load_command % i)
```

Comment:
Do you update `guest_load_command` definition in cfg?

Reply:
Thanks, updated it.

Reply:
@sitoliu Sorry, where do you update it?

Reply:
@vivianQizhu Please see https://github.com/avocado-framework/avocado-vt/pull/1706